dbm.opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
```

Comment:
Maybe Xavier should be marked as maintainer I guess.

Reply:
not sure he wants to be bothered